ffc-ahwr-event/function.json

```diff
@@ -28,6 +28,13 @@
       "type": "table",
       "direction": "out"
     },
+    {
+      "tableName": "ffcahwrexception",
```

Comment:
I think we could call this `ffcahwrineligibility` to indicate it is not generic for all exceptions and it is more for capturing known cases where a business is not eligible